src/azure-cli/azure/cli/command_modules/acr/_errors.py

```diff
@@ -117,6 +117,11 @@ def format_error_message(self, *args):
     "Access to registry '{}' was denied. Response code: {}. Please try running 'az login' again to refresh permissions."
 )
 
+CONNECTIVITY_REFRESH_TOKEN_TOOMANYREQUESTS_ERROR = ErrorClass(
+    "CONNECTIVITY_REFRESH_TOKEN_TOOMANYREQUESTS_ERROR",
+    "Too many requests, access to registry '{}' was denied. Response code: 429. Please wait a moment before trying again."
+)
```

Comment:
Should we have a generic `CONNECTIVITY_TOOMANYREQUESTS_ERROR`, which is not specific to refresh token? Just like `CONNECTIVITY_FORBIDDEN_ERROR`?